lib/internal/streams/operators.js

```diff
@@ -147,6 +151,37 @@ async function * map(fn, options) {
   }
 }
 
+async function some(fn, options) {
+  // https://tc39.es/proposal-iterator-helpers/#sec-iteratorprototype.some
+  // Note that some does short circuit but also closes the iterator if it does
+  const ac = new AbortController();
+  if (options?.signal) {
+    options?.signal?.addEventListener('abort', () => ac.abort(), {
```

Comment:
```suggestion
    options.signal.addEventListener('abort', () => ac.abort(), {
```

Reply:
@aduh95 is there any reason you're reviewing these but not approving/requesting changes on them? Are you afraid your LGTM will count as a TSC approval or something? (I am going to wait until I get an LGTM from Matteo and Robert on each of these PRs anyway most likely - but it would be great to have more people involved and while these small suggestions are good I'd also like your opinion on the APIs and substance of the PR)

Reply:
Sorry if those none approving reviews is giving you bad signals, that was not my intention. I've had some bad experience of approving/landing PRs on stream core modules that turned out to be detrimental and had to be reverted, so I try to be more cautious when giving my opinion on such PRs.
The PR is not semver-major, so no my TSC membership would not be relevant here I don't think; and since only the first two 2 approving reviews help the PR land, I was under the impression my explicit approval is not necessary.
I'm still reviewing the code to see if I see things that can be improved, but feel free to ignore if you disagree, or correct me when you think I'm wrong.

Reply:
> Sorry if those none approving reviews is giving you bad signals

Sorry, I meant kind of the opposite. I find the reviews helpful (no bad signals!), having more people look at this is a good thing! These are new APIs, they change (I hope!) how people work with the fundamental abstraction Node.js has for streaming data (files, sockets, everything!) and the more feedback the better.

What I meant is that while it's nice having this sort of feedback (like the unnecessary `?` here) I also want to hear what you think about the changes and new APIs.

> The PR is not semver-major, so no my TSC membership would not be relevant here I don't think; 

As far as I remember PRs that add new APIs to Node need TSC approval :)

Reply:
> (If you don't feel comfortable approving I still want to hear "this is a good change" or "this is a bad change imo")

That's fair, especially since I would probably say it if I was -0 or -1 on a change. I'm hyped about this iterator helper proposal, and I'm glad it's getting its way in Node.js already.
TL;DR imho this is a good change :)

> As far as I remember PRs that add new APIs to Node need TSC approval :)

I'm not sure this is the case, the collaborator guide requires TSC reviews only for semver-major changes: https://github.com/nodejs/node/blob/406e6d87c4375037ca52abb5750a3d847f03bc21/doc/guides/collaborator-guide.md#L470-L479

For semver-minor and semver-patch PRs, there is no requirement for reviews from TSC members: https://github.com/nodejs/node/blob/406e6d87c4375037ca52abb5750a3d847f03bc21/doc/guides/collaborator-guide.md#L853-L857

Maybe you were thinking about the rule for adding new _modules_ in core? https://github.com/nodejs/node/blob/406e6d87c4375037ca52abb5750a3d847f03bc21/doc/guides/collaborator-guide.md#L399-L404